src/antsibull_docs/augment_docs.py

```diff
@@ -60,6 +60,30 @@ def add_full_key(options_data: t.Mapping[str, t.Any], suboption_entry: str,
                 _full_keys=full_keys_k)
 
 
+def _add_seealso(seealso: t.List[t.MutableMapping[str, t.Any]],
+                 plugin_info: t.Mapping[str, t.Mapping[str, t.Any]],
+                 ) -> None:
+    for entry in seealso:
+        if entry.get('description'):
+            continue
+        plugin = ''
+        plugin_type = 'module'
+        if entry.get('module'):
```

Comment:
Would `if ... in ...` work here and in the rest of this change? Or are you also trying to check if e.g. `entry['module']` is falsy when it's defined?

Reply:
I'm also checking for falsy when it's defined.